assets/js/components/SocketHandler.jsx

```diff
@@ -83,24 +53,21 @@ class SocketHandler extends Component {
   render() {
     return (
       <div>
-        {this.props.children}
+        {React.cloneElement(this.props.children, { socket: this.socket, currentTeamId: this.props.currentTeamId })}
```

Comment:
how are these being accessed?

Reply:
also have you looked into the new context api? https://reactjs.org/docs/context.html